packages/boot/test/acceptance/controller.booter.app.acceptance.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright IBM Corp. 2013,2018. All Rights Reserved.
+// Node module: @loopback/boot
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Client, createClientForHandler, TestSandbox} from '@loopback/testlab';
+import {RestServer} from '@loopback/rest';
+import {BootOptions} from '@loopback/core';
+import {ControllerBooter, BootComponent} from '../../index';
+import {resolve} from 'path';
+
+describe('controller booter acceptance tests', () => {
+  // tslint:disable-next-line:no-any
+  let app: any;
+  const SANDBOX_PATH = resolve(__dirname, '../sandbox');
+  const sandbox = new TestSandbox(SANDBOX_PATH);
+  const bootConfig = {
+    projectRoot: SANDBOX_PATH,
+  };
+
+  beforeEach(resetSandbox);
+  beforeEach(getApp);
+
+  afterEach(stopApp);
+
+  it('binds controllers using ControllerDefaults and REST endpoints work', async () => {
+    await app.booter(ControllerBooter);
```

Comment:
I find this weird. Why are the consumers of `ControllerBooterApp` required to register individual booters used by the app? IMO, registering required booters is the responsibility of `ControllerBooterApp`.

Reply:
Good point. This can definitely be moved to be a part of `ControllerBooterApp`.